### PR TITLE
[lte][mme] Fix for MME config assert due to not null terminated strings

### PR DESCRIPTION
**Pull request by emakeev**
Signed-off-by: Evgeniy Makeev <evgeniym@fb.com>

## Summary

 Fix for MME config assert due to not null terminated strings

## Test Plan

unit/integration tests

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/c/oai/tasks/mme_app/mme_config.c [hunk 1]
```diff
@@ -958,9 +958,10 @@ int mme_config_parse_file(mme_config_t* config_pP) {
             // Convert to 3gpp PLMN (MCC and MNC) format
             // First 3 chars in astring is MCC next 3 or 2 chars is MNC
             memcpy(fed_mode_mcc, astring, MAX_MCC_LENGTH);
-            memcpy(
-                fed_mode_mnc, &astring[MAX_MCC_LENGTH],
-                (strlen(astring) - MAX_MCC_LENGTH));
+            fed_mode_mcc[MAX_MCC_LENGTH] = 0; // null terminated string
```
Comment (ulaskozat) [inline lte/gateway/c/oai/tasks/mme_app/mme_config.c]: it is a nit, but using ‘\0’ might be more informative.

File: lte/gateway/c/oai/tasks/mme_app/mme_config.c [hunk 2]
```diff
@@ -958,9 +958,10 @@ int mme_config_parse_file(mme_config_t* config_pP) {
...
+            n = strlen(astring) - MAX_MCC_LENGTH
+            memcpy(fed_mode_mnc, astring + MAX_MCC_LENGTH, n);
+            fed_mode_mnc[n] = 0; // null terminated string
```
Comment (ulaskozat) [inline lte/gateway/c/oai/tasks/mme_app/mme_config.c]: same as previous comment.

**Review by ulaskozat — APPROVED** [review]
LGTM.

File: lte/gateway/c/oai/tasks/mme_app/mme_config.c [hunk 3]
```diff
@@ -958,9 +958,10 @@ int mme_config_parse_file(mme_config_t* config_pP) {
             // Convert to 3gpp PLMN (MCC and MNC) format
             // First 3 chars in astring is MCC next 3 or 2 chars is MNC
             memcpy(fed_mode_mcc, astring, MAX_MCC_LENGTH);
-            memcpy(
-                fed_mode_mnc, &astring[MAX_MCC_LENGTH],
-                (strlen(astring) - MAX_MCC_LENGTH));
+            fed_mode_mcc[MAX_MCC_LENGTH] = 0; // null terminated string
+            n = strlen(astring) - MAX_MCC_LENGTH
```
Comment (ulaskozat) [inline lte/gateway/c/oai/tasks/mme_app/mme_config.c]: missing `;`

**Review by ulaskozat — CHANGES_REQUESTED** [review]
missing ';'

File: lte/gateway/c/oai/tasks/mme_app/mme_config.c (hunk 1 above)
Comment (uri200) [reply lte/gateway/c/oai/tasks/mme_app/mme_config.c:961]: do we need the same for `fed_mode_mnc`?

**Review by ulaskozat — APPROVED** [review]
LGTM

**Pull request merged**